nautobot_ssot_infoblox/diffsync/adapters/infoblox.py

```diff
@@ -59,6 +59,7 @@ def load_prefixes(self):
                 description=_pf.get("comment", ""),
                 status=_pf.get("status", "active"),
                 ext_attrs=get_ext_attr_dict(extattrs=_pf.get("extattrs", {})),
+                vlans=build_vlan_map(vlans=_pf["vlans"]) if _pf.get("vlans") else {},
```

Comment:
`_pf["vlans"]` this looks to me like a potential `KeyError` candidate. You are actually handling it with a `get` in the second part

Reply:
you are right